CIP-_/README.md

```diff
@@ -0,0 +1,209 @@
+---
+CIP: "?"
+Title: Optimized Coin Selection Algorithms for Cardano
+Category: Wallets
+Authors:
+    - Nicholas Maselli <maselli.nicholas@gmail.com>
+Implementors: CardanoSharp 7, SaturnNFT, Levvy Finance
+Discussions:
+Status: Pending
```

Comment:
```suggestion
Status: Proposed
```
... effectively meaning the same thing ("pending" assignment of `Active` status)